src/js/index.js

```diff
@@ -0,0 +1,130 @@
+import createHeader from './header';
```

Comment:
Keeping all files in the same folder is generally a bad practice. It may work for such a small project, but since it has educational purpose, please choose an approach for folder structure that you think will work best and implement it here.

Reply:
Created components folder, and there created header, main and footer folders for js modules.

Reply:
Currently you have **components** folder that includes **constants**, **createHtml** and **main** folders. Please see my comments below:

1. Move **constants** out of **components**, it's not a component.
2. rename **createHtml** to have the same name as the component you store in it.
3. rename **main** to have the same name as the component you store in it.

Reply:
done